src/lib/db.js

```diff
@@ -128,6 +128,19 @@ function * truncateTables () {
   }
 }
 
+function * isConnected () {
+  const query = knex.client.config.client === 'strong-oracle'
+    ? 'SELECT 1 FROM DUAL' : 'SELECT 1'
+  return knex.raw(query)
+  .then(() => {
+    return true
+  })
+  .catch((e) => {
+    console.log('DB error: ', e)
```

Comment:
I think you have to use bunyan format logging: https://github.com/interledger/five-bells-ledger/blob/master/package.json#L15